documentation/rfc/doc-routing-algorithm.md

```diff
@@ -0,0 +1,148 @@
+
+Basically, you give to RAPTOR the set of journey patterns it can use, the set of stop points it can use (for transfers), the datetime you can reach the stop points and the direction. It gives you, for each stop points and each number of transfers, the earliest arrival datetime.
+
+Our implementation is very close to the original RAPTOR algorithm. There is 2 added functionnalities:
+ - stay in: they are oportunistic, i.e. it is not guananteed to be optimally using the stay in. The journey patterns doesn't take the stay in into account.
+ - ITL: as we are only one zone per stop time, the implementation is quite straightforward: at the furst pickup, we set the current zone to the zone of the stop point. If the zone change, no more drop off is forbidden because of the zone because we can choose afterward the good pick up.
+ 
+ The journey pattern are generated automatically. They must have the same succession of stop times (except datetimes) and must not overtake.
+
+### The first pass on example 1
+
+In this section, you can find some examples of the result of raptor. Our implementation doesn't remember the vehicle journeys used, they are chosen by the raptor solution reader, explained later.
+
+The input of the algorithm is a set of journey patterns and stop points that can be used, and a starting datetime plus durations to access the reachable stop points. These 2 last inputs are used to compute the first TR0 line.
+
+The PTx lines correspond to the earliest arrival to a stop point using x vehicles (including stay in). The TRx correspond to the earliest arrival to a stop point after x vehicles plus the connections.
+
+Starting from A at 7:45:
+
+|level| A  | B  | C  | D  | E  | F  |  G      |
+|-----|----|----|----|----|----|----|---------|
+|TR0  |7:45|    |    |    |    |    |         |
+|PT1  |    |8:30|8:33|8:36|8:50|    |         |
+|TR1  |    |8:32|8:35|8:38|8:52|    |         |
+|PT2  |    |    |    |    |    |9:00|**10:00**|
+|TR2  |    |    |    |    |    |9:03|10:02    |
+|PT3  |    |    |    |    |    |    |         |
+
+Starting from A at 7:55:
+
+|level| A  | B  | C  | D  | E  | F  |  G      |
+|-----|----|----|----|----|----|----|---------|
+|TR0  |7:55|    |    |    |    |    |         |
+|PT1  |    |8:30|8:33|8:36|9:10|    |         |
+|TR1  |    |8:32|8:35|8:38|9:12|    |         |
+|PT2  |    |    |    |    |9:00|9:00|**10:20**|
+|TR2  |    |    |    |    |9:02|9:03|10:22    |
+|PT3  |    |    |    |    |    |    |**10:00**|
+|TR3  |    |    |    |    |    |    |10:02    |
+|PT4  |    |    |    |    |    |    |         |
+
+The different PTx on the destination stop points gives the earliest arrival to our destinations for each number of connection. In this case, if we target G, starting at 7:45 from A gives G at 10:00using 0 connection, but starting at 7:55 gives 12:00 using 0 connection and 10:00 using 1 connection.
```

Comment:
I don't think I get your point here. Also; that there is no 12:00 stop time in the example.